docs/content/how-to/powervs/create-cluster-powervs.md

```diff
@@ -48,8 +48,19 @@ where
 
 Running this command will create [infra](create-infra-separately.md) and manifests for the hosted cluster and deploys it.
 
-## Utilizing Power Edge Router(PER) via Transit Gateway
-To use IBM Cloud's PER feature via transit gateway, need to pass `--use-power-edge-router` and `--transit-gateway-location $TRANSIT_GATEWAY_LOCATION` flags to create cluster command like below.
+## Using Transit Gateway (Default Networking Mode)
```

Comment:
IMO this block is not required, let's update the previous block itself with `--transit-gateway-location` flag and explaining it like other flags would be enough. By keeping it in separate block will confuse the user.